packages/api/src/routes/ehr/athenahealth/chart.ts

```diff
@@ -1,35 +1,34 @@
 import Router from "express-promise-router";
 import httpStatus from "http-status";
 import { Request, Response } from "express";
-import { writeMedicationToChart } from "../../../external/ehr/athenahealth/command/write-to-chart";
+import { writeMedicationToChart } from "../../../external/ehr/athenahealth/command/write-medication-to-chart";
+import { writeConditionToChart } from "../../../external/ehr/athenahealth/command/write-condition-to-chart";
+import { writeVitalsToChart } from "../../../external/ehr/athenahealth/command/write-vitals-to-chart";
 import { requestLogger } from "../../helpers/request-logger";
 import { asyncHandler, getCxIdOrFail, getFrom, getFromQueryOrFail } from "../../util";
-import { getAuthorizationToken } from "../../util";
 import { handleParams } from "../../helpers/handle-params";
 
 const router = Router();
 
 /**
  * POST /ehr/athenahealth/chart/:id/medication
  *
- * Tries to retrieve the matching Metriport patient
+ * Tries to write the medication to the patient's chart
```

Comment:
nit: why "tries to..."? That sounds like it will silently fail and that it's expected to fail sometimes.

Reply:
Changed.